test/jdk/java/lang/System/SecurityManagerWarnings.java

```diff
@@ -72,8 +71,8 @@ public static void main(String args[]) throws Exception {
         }
     }
 
-    // When -Djava.security.manager is not set, or set to "allow",
```

Comment:
The comment was incorrect. Setting "allow" now causes the JVM to fail with an error.